File: ViviamAcuna_GenerarTiempos.cpp
#include <iostream>
#include <ctime>

using namespace std;

//Se declaran las funciones
int fibonacci(int N);
float get_time(int N);
//Funcion de fibonacci
int fibonacci(int N){
  if (N== 0 || N==1 ){
    
    return N;
  }
  
  else{
    return fibonacci(N-2)+ fibonacci(N-1);
  }
 }
//Calculo de tiempo

float get_time(int N){
  clock_t t;
  t = clock();
// SOME CODE THAT TAKES TIME
  fibonacci(N);
  t = clock() - t;
  float time = ((float)t)/CLOCKS_PER_SEC;
  return time;

}
//Funcion main

int main(){

  int N;
  N=32; 
  fibonacci(N);
  get_time(N);
  cout << get_time(N) << endl;

  return 0;
}

File: ViviamAcuna_GenerarTiempos.py
import numpy as np
import time

N=36
#Funcion de Fibonacci
def fibonacci(N):
    if (N==0 or  N==1):
        return N
    else:
        return fibonacci(N-1) + fibonacci(N-2)
#Funcion que da el tiempo
def get_time(N):
    t0 = time.time()
    
# SOME CODE THAT TAKES TIME
    fibonacci(N)
    t1 = time.time()-t0
    return t1
for i in range(N):
    print i,get_time(i)
